server/application/application.proto

```diff
@@ -280,6 +280,7 @@ message LinksResponse {
 message ListAppLinksRequest {
 	required string name = 1;
 	optional string namespace = 3;
+	optional string appNamespace = 4;
```

Comment:
I am curious why we need a new field when we can just use `namespace` which already exists. I think the bug is that the UI does not actually use the namespace request param at all when fetching the links if we just pass the namespace in the ui request it should fix it imo.

Reply:
You are right.

I used `appNamespace` to be consistent with other endpoints such as `ApplicationUpdateSpecRequest`, `ApplicationPatchRequest` `ApplicationSyncRequest`...

If you prefer I can use `namespace` field

Reply:
Because we already have the field I'd prefer to use that as it would mean no new backend changes just a small ui fix.

Reply:
Doing the fix now!

Reply:
All done @gdsoumya thanks a lot for the review